package.json

```diff
@@ -1,18 +1,23 @@
 {
   "name": "use-http",
-  "version": "0.1.22",
-  "main": "src/main.js",
+  "version": "0.1.23",
```

Comment:
Bumping the version.

Reply:
Can you explain this?  When importing the package, where will the files come from if we don't have a `"main": "file.js"`

Reply:
Sorry, I don't know why github is showing this as removed, it's line 4 of the package.json. I changed it to: `"main": "build/main.js"` so that the main entry point of the npm package is the transpiled `main.js` file.

Reply:
I will merge this once I get home. Just on vacation